app/controllers/data_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class DataController < ApplicationController
```

Comment:
@CloCkWeRX I'd like your opinion on this. It's holding the repeating code from the garden, harvest, seed, and plantings controllers. Putting an end to typos in one controller making iit inconsistent.
It could be a mixin instead - but an inherited controller methods seems more railsy. not sure tho.

Reply:
I think it makes sense. If we had two competing concerns 'controller with slugs' and 'controller with xyz'; that'd be the point to refactor to a mixin.